lib/cf_app_discovery/service_broker.rb

```diff
@@ -54,9 +54,13 @@ class ServiceBroker < Sinatra::Application
     delete "/v2/service_instances/:instance_id/service_bindings/:id" do |_, binding_id|
       content_type :json
       service_binding = client.service_binding(binding_id)
-      app_guid = service_binding.fetch(:entity).fetch(:app_guid)
-      cleaner = Cleaner.new(filestore_manager: filestore_manager)
-      cleaner.remove_old_target(app_guid)
+      if service_binding.has_key?(:entity) == false
```

Comment:
can we flip this `if` to put the positive case first?  it's confusing to say `if x == false then foo else bar` rather than `if x then bar else foo`

Reply:
Sure, will update